### ISSUE TITLE
In Right-to-Left languages, the launcher does not correctly centre on screen.

### ISSUE THREAD

**Issue by XV-02** [pop-os/shell#1384]

**Comment by dabasdev** [comment]
@jacobgkau  
if i understand , the problem not solved until this time?

**Comment by jacobgkau** [comment]
> @jacobgkau if i understand , the problem not solved until this time?

Yes, the issue being open indicates the problem still exists. The issue would be closed if the problem was solved.

**Comment by dabasdev** [comment]
@jacobgkau   i would to notice that if i change the sys language from arabic to english , problem fix
